.travis.yml

```diff
@@ -6,10 +6,15 @@ sudo: false
 before_script:
   bundle exec rake db:create db:migrate
   
-deploy:
-provider: heroku
-api_key:
-secure: "278ae214b4f4d40eed4ae2cb4035c72456b525b71779f1f794d0e41c7fe19aca81992cc7682301d233a6aa1d60c1f6c911a37a952fe8e765769e21ed4cf1e9c4"
+
+  deploy:
+  provider: heroku
+  api_key:
+  secure: 278ae214b4f4d40eed4ae2cb4035c72456b525b71779f1f794d0e41c7fe19aca81992cc7682301d233a6aa1d60c1f6c911a37a952fe8e765769e21ed4cf1e9c4
```

Comment:
Does this key need to be a string? It was in the original version. Travis says this is an "invalid yaml file."

Reply:
oh good catch i wil fix it